src/step-definitions/hooks.js

```diff
@@ -1,36 +1,39 @@
 const { After } = require("@wdio/cucumber-framework");
 const Boards = require("../pageobjects/pages/boards.page");
-const LoginPage = require("../pageobjects/pages/login.page");
-const ProfilePage = require("../pageobjects/pages/profile.page");
+const HomePage = require("../pageobjects/pages/home.page")
+const LogoutPage = require("../pageobjects/pages/logout.page")
+// const LoginPage = require("../pageobjects/pages/login.page");
+// const ProfilePage = require("../pageobjects/pages/profile.page");
 
-const logout = async () => {
-  await ProfilePage.profileIcon.click();
-  const logoutButton = await LoginPage.logoutBtn;
 
-  await logoutButton.waitForDisplayed({ timeout: 3000 });
+const logout = async () => {
+await HomePage.header.profileIcon.click();
+  const logoutButton = await HomePage.memberMenu.logoutBtn;
+  await logoutButton.waitForDisplayed({ timeout: 5000 });
   await logoutButton.click();
-  const logoutSubmitButton = await LoginPage.logoutSubmitBtn;
-  await logoutSubmitButton.waitForClickable({ timeout: 2000 });
+  const logoutSubmitButton = await LogoutPage.logoutForm.logoutSubmitBtn;
+  await logoutSubmitButton.waitForClickable({ timeout: 5000 });
   await logoutSubmitButton.click();
 };
 After({ name: "logout user", tags: "@logout" }, async () => {
   await logout();
 });
 
 After({ name: "removing list", tags: "@removeList" }, async () => {
-  const listEditButtons = await Boards.listEditBtn;
-  for (let btn of listEditButtons) {
-    await btn.click();
-    await Boards.archiveListBtn.waitForDisplayed({ timeout: 2000 });
-    await Boards.archiveListBtn.click();
-  }
+  // const listEditButtons = await Boards.listEditBtn;
+  // for (let btn of listEditButtons) {
+  //   await btn.click();
+  //   await Boards.archiveListBtn.waitForDisplayed({ timeout: 2000 });
+  //   await Boards.archiveListBtn.click();
+  // }
```

Comment:
in code we shouldn't have commented lines (only in exceptional cases)
it you don't need this hook - remove it